torch/csrc/jit/codegen/cuda/ir_utils.h

```diff
@@ -172,10 +172,13 @@ Val* replaceValInIndexVal(
     Val* index,
     const std::unordered_map<Val*, Val*>& replacement_map);
 
-// Makes rfactor generic with reduction ops and Welford
+// Makes rfactor generic with reduction ops and Welford.
+// Note that the axes must be positions excluding broadcast. For example,
+// if you have a tensor T1[b, rS1, rS2, rS3], and you want to rFactor axis rS2
+// and rS3, you will need to use rfactorHelper(T1, {1, 2}) instead
```

Comment:
Is this change really necessary? I think it makes most sense to have the same parameter semantics as `reductionOp`.

Reply:
`rfactorHelper` converts the position parameters with no broadcast to the raw positions, so can it be done by the scheduler instead?

Reply:
I agree, will change

Reply:
Changed. It is much cleaner now. Thank you for your suggestion.